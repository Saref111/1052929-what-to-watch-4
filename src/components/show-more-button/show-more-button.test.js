import React from "react";
import renderer from "react-test-renderer";
import ShowMoreButton from "./show-more-button.jsx";

describe(`ShowMoreButton should match snapshots`, () => {
  test(`ShowMoreButton should render`, () => {
    const tree = renderer.create(<ShowMoreButton
      onShowMoreButtonClickHandler={() => {}}
      isShown={true}
    />);

    expect(tree).toMatchSnapshot();
  });

  test(`ShowMoreButton shouldn't render`, () => {
    const tree = renderer.create(<ShowMoreButton
      onShowMoreButtonClickHandler={() => {}}
      isShown={false}
    />);

    expect(tree).toMatchSnapshot();
  });
});

