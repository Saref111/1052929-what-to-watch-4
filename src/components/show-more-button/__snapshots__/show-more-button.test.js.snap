// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShowMoreButton should match snapshots ShowMoreButton should render 1`] = `
<div
  className="catalog__more"
>
  <button
    className="catalog__button"
    onClick={[Function]}
    type="button"
  >
    Show more
  </button>
</div>
`;

exports[`ShowMoreButton should match snapshots ShowMoreButton shouldn't render 1`] = `""`;
